d2l-outcomes-coa-eval-override.js

```diff
@@ -163,100 +162,116 @@ export class d2lOutcomesCOAEvalOverride extends LocalizeMixin(LitElement) {
 		`;
 	}
 
-	render() {
+	renderElementHeading() {
```

Comment:
Internal functions should be `_` prefixed
```suggestion
	_renderElementHeading() {
```